test/testdata/profile.json

```diff
@@ -0,0 +1,4 @@
+{
+	"name": "Lee Sedol",
+	"game": "Go"
```

Comment:
I would swap the order of these two, as Go marshals maps into json in key-alphabetical order.

Reply:
ok.